### PR TITLE
SX: Automatic vendor prefixes

### PR DESCRIPTION
**Pull request by michalsanger**
Vendor prefixes are automatically added into the generated CSS.

### CODE REVIEW

File: src/sx/src/injectRuntimeStyles.js [hunk 1]
```diff
@@ -82,16 +83,25 @@ export default function injectRuntimeStyles(styleBuffer: StyleBufferType) {
     if (node instanceof StyleCollectorNode) {
       if (hasStyleRule(matchFunction(node)) === false) {
         // apply missing styles
-        styleSheet.insertRule(node.printNodes().join(''), insertIndex);
+        const rule = serialize(
+          compile(node.printNodes({ trailingSemicolon: true }).join('')),
```
Comment (tbergq) [inline src/sx/src/injectRuntimeStyles.js:87]: Can't comment on that line, but you probably need this for line 62 as well, inject runtime keyframes.

Comment (michalsanger) [reply src/sx/src/injectRuntimeStyles.js:90]: Done

**Review by tbergq — APPROVED** [review]

File: src/sx/src/injectRuntimeStyles.js [hunk 2]
```diff
@@ -58,7 +59,10 @@ export function injectRuntimeKeyframes(css: string, name: string) {
   };
 
   if (hasStyleRule(matchFunction) === false) {
-    styleSheet.insertRule(css, styleSheet.cssRules.length);
+    const rules = compile(serialize(compile(css), middleware([prefixer, stringify]))) ?? [];
+    rules.forEach((rule) => {
+      styleSheet.insertRule(serialize([rule], stringify), styleSheet.cssRules.length);
```
Comment (mrtnzlml) [inline src/sx/src/injectRuntimeStyles.js:64]: I don't want to discourage you from these efforts but at the same time, I don't want to leave this without mentioning my concern: isn't `serialize(compile(serialize(compile(css))))` a bit too much? 🤔 Have you thought about this @michalsanger?

Comment (michalsanger) [reply src/sx/src/injectRuntimeStyles.js:66]: Yes that was ugly, the api of Stylis is a little bit strange. Please check again.

**Review by mrtnzlml — APPROVED** [review]

**Pull request merged**